include/MainWindow.h

```diff
@@ -149,6 +187,8 @@ public slots:
 	void toggleWindow( QWidget *window, bool forceShow = false );
 	void refocus();
 
+	bool m_recoverSession;
+	bool m_limitedSession;
```

Comment:
I started out with recoverSession and then added 'limited' along the way. Maybe this should be done with an enum instead, something like:
enum Session { Normal, Recover, Limited }; ?